jukebox_radio/streams/models/queue_model.py

```diff
@@ -34,16 +34,27 @@ def serialize(self, queue, is_child=False):
         try:
             queue_children = queue.ordered_children
         except AttributeError:
+            # TODO: we should fetch children here, but as of now, no case
+            #       justifies writing this code.
             queue_children = []
         children = []
         for child in queue_children:
-            children.append(self.serialize(child, is_child=True))
+            children.append(self.serialize(child))
         obj["children"] = children
 
         try:
             active_intervals = queue.active_intervals
         except AttributeError:
-            active_intervals = []
+            active_intervals = (
```

Comment:
bugfix here. technically I am repeating myself here, the code should probably be condensed. Long story short: I believe I would have to tear up a lot of ground just have ensure a shared code path. So for now I am just leaving the duplication.

https://github.com/0x213F/jukebox-radio-django/blob/main/jukebox_radio/streams/models/queue_model.py#L148